### PR TITLE
Added proposal

### PR DESCRIPTION
**Pull request by Dmilut**
Added proposal from group 5

### CODE REVIEW

**Review by nikhilshahdemoaccount — COMMENTED** [review]
This looks to be really a great project domain, 

looking forward to see it in action, 

however if you can provide more details on what are the dimensions, how will you be collecting the data, from which all sources, what kind of data can be visualized



Kindly elaborate More on this

File: P2-Proposals/synopsis.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+0.1
+## Context and Goals
+Based on open data sources, the project offers opportunities for collecting, aggregating, transforming, analyzing and visualizing statistics on the incidence of COVID-19 in the United States.
+## Technical solution overview
+SQL Server
+SSIS
+SSAS
+SSRS
+### Architecture
+...
```
Comment (kaurrevature) [inline P2-Proposals/synopsis.md]: Please add the architecture

**Review by kaurrevature — CHANGES_REQUESTED** [review]

File: P2-Proposals/synopsis.md (hunk 1 above)
Comment (Dmilut) [reply P2-Proposals/synopsis.md]: Fixed

**Comment by Dmilut** [comment]
Updated

**Comment by Dmilut** [comment]
I updated the proposal, please check it

**Comment by kaurrevature** [comment]
Good work @Dmilut , it is looking great!!

**Pull request merged**